tests/integration/suite/webhooks-ledger-committed-transaction.go

```diff
@@ -0,0 +1,89 @@
+package suite
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+
+	"github.com/formancehq/formance-sdk-go"
+	. "github.com/formancehq/stack/tests/integration/internal"
+	webhooks "github.com/formancehq/webhooks/pkg"
+	"github.com/formancehq/webhooks/pkg/security"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Given("An environment configured with a webhook sent on created transaction", func() {
+	var (
+		httpServer *httptest.Server
+		called     chan struct{}
+		secret     = webhooks.NewSecret()
+	)
+
+	BeforeEach(func() {
+		called = make(chan struct{})
+		httpServer = httptest.NewServer(
+			http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				defer close(called)
```

Comment:
Not a big deal, but you already check the signature in another test, you could omit it.

Reply:
You're right, just removed it